web_client/app/infrastructure/api/api.ts

```diff
@@ -19,7 +19,11 @@ class API {
     if (method != 'GET' && body) {
       request.body = JSON.stringify(body);
     }
-    return (fetch as any)(this.apiURL + url, request);
+    if (url.includes('://')) {
+      return (fetch as any)(url, request)
```

Comment:
I assume we'll want to make a new file for the external API?

Reply:
I dunno that we need a whole new API class and file, but you're right that we definitely we don't want to do this, which sends the users' credentials to whatever URL gets plopped in there.

I'll take a look.